gpu-operator/google-gke.rst

```diff
@@ -30,21 +30,21 @@ NVIDIA GPU Operator with Google GKE
 About Using the Operator with Google GKE
 ****************************************
 
-You can use the NVIDIA GPU Operator with Google Kubernetes Engine (GKE),
-but you must use an operating system that is supported by the Operator.
+There are two ways to use NVIDIA GPU Operator with Google Kubernetes Engine (GKE):
 
-By default, Google GKE configures nodes with the Container-Optimized OS with Containerd from Google.
-This operating system is not supported by the Operator.
+- Use the default operating system, Container-Optimized OS.
 
-To use a supported operating system, such as Ubuntu 22.04 or 20.04, configure your
-GKE cluster entirely with Ubuntu containerd nodes images or with a node pool
-that uses Ubuntu containerd node images.
+  GKE configures the nodes with containerd from Google and the Google driver installer manages the NVIDIA GPU Driver.
+  The GPU Operator manages the lifecycle and upgrade of the remaining software components such as the device plugin, container toolkit, DCGM, and so on.
 
-By selecting a supported operating system rather than Container-Optimized OS with Containerd,
-you can customize which NVIDIA software components are installed by the GPU Operator at deployment time.
-For example, the Operator can deploy GPU driver containers and use the Operator
-to manage the lifecycle of the NVIDIA software components.
+- Use Ubuntu operating system with containerd.
 
+  You can choose to use the Google driver installer to manage the NVIDIA GPU Driver.
+  Alternatively, you can use the Operator and NVIDIA Driver Manager to manage the driver lifecycle and upgrades.
```

Comment:
From the NVIDIA perspective, we recommend the latter approach on Ubuntu (have GPU Operator manage every software component) -- I am wondering if we should make that more evident here and in the procedure for Ubuntu.